Python/cp/Knuth_Morris_Pratt.py

```diff
@@ -0,0 +1,48 @@
+# Implementation of Knuth Morris Pratt Algorithm in Python
+def KMP_Search(pattern, text):
+    a = len(text)
+    b = len(pattern)
+    prefix = prefix_arr(pattern, b)
+  
+    initial = []
+    m = 0
+    n = 0
+  
+    while m != a:
```

Comment:
Use meaningful and easy to understand variable names.

Reply:
For this algorithm this variables are meaningful variables